Uppsala/UppEt21.xml

```diff
@@ -0,0 +1,815 @@
+                                            <locus from="104r" to="113v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q12" n="11">
+                                            <num value="11">፲፩</num>
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="114r" to="123v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q13" n="12">
+                                            <num value="12">፲፪</num>
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="124r" to="133v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q14" n="13">
+                                            <num value="13">፲፫</num>
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="134r" to="143v"></locus>
+                                        </item>
+                                        
+                                    </list>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="30 32">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">130</citedRange>
+                                </bibl> calls it a "middle-size even hand" (<foreign xml:lang="de">Mittelgroße gleichmäßige Hand</foreign>).</desc>
+                                <date notBefore="1700" notAfter="1799">18th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">Paper sticker on the inside of the front board.</desc>
+                                    <q xml:lang="sv">
+                                        Uppsala Universitets<lb/>
+                                        Bibliotek<lb/>
+                                        Handskr. avd.<lb/>
+                                        O Etiop. 21<lb/>
+                                    </q>
+                                    <note>Below the rest of the text, a note in pencil: "Löfgren nr 38"</note>
```

Comment:
To my opinion, this note deserves its own extra.

Reply:
well seen! I agree

Reply:
Ok! I'll change it